lib/DDG/Spice/Meetup.pm

```diff
@@ -0,0 +1,62 @@
+package DDG::Spice::Meetup;
+# ABSTRACT: This Spice was inspired by an anonymous suggestion in the Instant Answers Forum
+# # # # # # source: https://duck.co/ideas/idea/1126/already-an-instant-answers-for-meetup
+# # # # # # When a user searches for "meetup + city name" they should receive
+# # # # # # instant answers of suggested meetups in that city sorted by the number of
+# # # # # # members in the group.
+
+use DDG::Spice;
+use YAML::XS 'LoadFile';
+use Text::Trim;
+
+spice is_cached => 1;
+
+name "Meetup";
+source "Meetup";
+icon_url "https://icons.duckduckgo.com/ip2/www.meetup.com.ico";
+description "Show meetups for a given city";
+primary_example_queries "meetups in philadelphia";
+secondary_example_queries "meetup groups near me";
+category "random";
+topics "social", "everyday", "travel";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Meetup.pm";
+attribution github => ["alohaas", "Olivia Haas"],
+            twitter => "livhaas";
+
+spice from => '([^/]+)/?(?:([^/]+)/?(?:([^/]+)|)|)';
+spice to => 'https://api.meetup.com/2/groups?&key={{ENV{DDG_SPICE_MEETUP_APIKEY}}}&country=$2&city=$1&state=$3&radius=50.0&page=20&order=members&callback={{callback}}';
```

Comment:
I guess since we don't have our geocoder exposed to the Spices (the thing that makes sense of the string "San francisco") I guess we have two options here:

- Only trigger on "meetups near me" and the like
- See if Meetup.com API can accept a free form string such as "San Francisco" without having the need to specify the state and the country (which is how https://duckduckgo.com/?q=weather+in+los+angeles&ia=weather works)